.github/workflows/roman_ci.yml

```
@@ -91,8 +43,9 @@ jobs:
         DD_GIT_REPOSITORY_URL: ${{ github.repositoryUrl }}
         DD_GIT_COMMIT_SHA: ${{ github.sha }}
         DD_GIT_BRANCH: ${{ github.ref_name }}
-      cache-path: ${{ needs.data.outputs.data_path }}
-      cache-key: data-${{ needs.data.outputs.data_hash }}-${{ needs.data.outputs.crds_context }}
+      cache-path: ${{ needs.data.outputs.crds_path }}
```

shouldn't this be the generic data path?
```suggestion
      cache-path: ${{ needs.data.outputs.data_path }}
```